CPS-XXXX/README.md

```diff
@@ -0,0 +1,76 @@
+---
+CPS: ?
+Title: Ouroboros not accounting for typical global network delays causes centralisation
+Status: Open
+Category: ?
+Authors:
+  - Terminada
+Proposed Solutions: []
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/943
```

Comment:
```suggestion
  - https://github.com/cardano-foundation/CIPs/pull/943
  - https://forum.cardano.org/t/problem-with-increasing-blocksize-or-processing-requirements/140044
```
This field is also intended for community discussions, if technically significant, so feel free to add this link if there is any ensuing technical discussion there.